### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by shlok30**
Hi Team,

Can you please review my product listing page.
The site is not responsive yet, I'll work on that later.

Link - https://jolly-keller-be11b2.netlify.app/product-page.html

Thanks in advance

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
 .cards-preview .img-container{
```
Comment (tushargoyal812) [inline styles.css]: suggestion: add a space before { its a good practice and use beautify extension in vs code it makes all the spacing correct.

**Review by tushargoyal812 — COMMENTED** [review]
only take care what i suggested and looks good to merge

File: styles.css (hunk 1 above)
Comment (shlok30) [reply styles.css]: Understood, thanks

File: product-page.html [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                </div>
+                <div class = "padding-s-block">
+                    <input type = "checkbox"> Womens Clothing</input>
+                </div>
+           </div>
+
+           <div class= "padding-m m2-top">
+               <div class="text-s bold">Rating</div>
+            <div class = "padding-s-block m2-top flex">
+                <input type = "radio" class = "radio-btn center-text" >
```
Comment (irajbhinde) [inline product-page.html]: add name & value in all the radio buttons, if you check your deploy, you are able to select all the radio buttons

File: product-page.html [hunk 3]
```diff
@@ -0,0 +1,199 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel = "stylesheet" href = "https://water-style-ui.netlify.app/main.css">
+    <link rel = "stylesheet" href = "styles.css">
```
Comment (irajbhinde) [inline product-page.html:8]: When you start working on further pages, you can create a main stylesheet consisting of navbar & most used styles, which you'll be using on all the pages.

File: product-page.html [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+    />
+    <title>Products</title>
+</head>
+<body>
+
+    <nav class = "navigation flex gap-xl align-center full-width">
```
Comment (irajbhinde) [inline product-page.html:17]: Using utility is good but align-center & full width is unnecessary

File: product-page.html [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+
+           <div class = "padding-m m2-top">
+               <div class = "text-s bold">Price</div>
+               <div>
+                    <input type="range" min="1" max="100" value="50" class = "slider m2-top full-width">
+               </div> 
+           </div>
+
+           <div class = "padding-m m2-top">
+                <div class = "text-s bold">Category</div>
```
Comment (irajbhinde) [inline product-page.html:44]: text-s is a class not easy to understand, you can use a better name here

File: styles.css [hunk 6]
```diff
@@ -16,4 +17,14 @@
...
     border-top-right-radius: 1rem;
+}
+
+.side-bar{
+    border:1px solid black;
+    flex-basis:15rem;
+} 
+
+.radio-btn{
+    height:20px;
```
Comment (irajbhinde) [inline styles.css]: you can use relative unit like rem instead of px

**Review by irajbhinde — COMMENTED** [review]
Few Suggestions - 

1. remove border radius from nav bar
2. add an anchor link with your index.html on the Water-ecom title with cursor pointer
3. add cursor pointer to cart & wishlist along with anchor links
4. add cursor pointer to clear 

Also look into single line feedbacks. You're doing good, keep up the hard work!

File: product-page.html (hunk 2 above)
Comment (shlok30) [reply product-page.html]: Understood, thanks

File: product-page.html (hunk 3 above)
Comment (shlok30) [reply product-page.html:8]: Cool

File: product-page.html (hunk 5 above)
Comment (shlok30) [reply product-page.html:44]: So for text I have made classes like text-xs,text-s,text-m and so on

Comment (shlok30) [reply product-page.html:44]: Thanks for the suggestion though

File: styles.css (hunk 6 above)
Comment (shlok30) [reply styles.css]: Understood, thanks

**Pull request merged**